src/beta_inc.jl

```diff
@@ -825,3 +825,125 @@ function beta_inc(a::T, b::T, x::T, y::T) where {T<:Union{Float16, Float32}}
 end
 beta_inc(a::Real, b::Real, x::Real, y::Real) = beta_inc(promote(float(a), float(b), float(x), float(y))...)
 beta_inc(a::T, b::T, x::T, y::T) where {T<:AbstractFloat} = throw(MethodError(beta_inc,(a, b, x, y,"")))
+
+
+#find x ; given p, q, Ix(p,q) = alpha, beta = logabsbeta(p,q)[1]
```

Comment:
keep the symbols consistent with what we had above, i.e.
```math
p = I_x(a,b)
p+q = 1
x+y = 1
```
where `I_x(a,b)` is defined as in https://dlmf.nist.gov/8.17.2

Reply:
yeah sure! I was going to change that.